build_tools/benchmarks/generate_benchmark_comment.py

```diff
@@ -4,51 +4,34 @@
 # Licensed under the Apache License v2.0 with LLVM Exceptions.
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-"""Posts benchmark results to GitHub as pull request comments.
+"""Generates benchmark results as pull request comments.
 
-This script is meant to be used by Buildkite for automation. It requires the
-following environment to be set:
+This script is meant to be used by CI. It requires the following environment
+variables to be set:
 
-- BUILDKITE_BUILD_NUMBER: the build number of current Buildkite build.
-- BUILDKITE_BUILD_URL: the link to the current Buildkite build.
-- BUILDKITE_COMMIT: the pull request HEAD commit.
-- BUILDKITE_PULL_REQUEST: the current pull request number.
-- GITHUB_TOKEN: personal access token to authenticate against GitHub API;
-    it should have "public_repo" and "gist" scope.
-
-if --query-base in toggled on, then it additionally requires:
-
-- BUILDKITE_PULL_REQUEST_BASE_BRANCH: the targeting base branch.
 - IREE_DASHBOARD_URL: the url to IREE's performance dashboard.
 
 This script uses pip package "markdown_strings".
-
-Example usage:
-  # Export necessary environment variables:
-  export ...
-  # Then run the script:
-  python3 post_benchmarks_as_pr_comment.py <benchmark-json-file>...
-  #   where each <benchmark-json-file> is expected to be of format expected
-  #   by BenchmarkResults objects.
 """
 
+import sys
+import pathlib
+
+# Add build_tools python dir to the search path.
+sys.path.insert(0, str(pathlib.Path(__file__).parent.with_name("python")))
+
+from typing import Any, Dict, Optional, Set, Tuple
```

Comment:
Reverted the change.

The example below shows the pattern to have "from ..." first then "import ...". I guess there is no clear guideline on their places?

Having `from...` on top does make it easier to sort them because "from..." is naturally in front of the "import...".

Reply:
Which example below? I think in the style guide they're consistently sorted by package name

Reply:
Ah, I see what the style guide means. Reordered all import in lexicographical order.